go.mod

```diff
@@ -484,7 +484,7 @@ replace (
 	gopkg.in/tomb.v1 => gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7
 	gopkg.in/warnings.v0 => gopkg.in/warnings.v0 v0.1.1
 	gopkg.in/yaml.v2 => gopkg.in/yaml.v2 v2.4.0
-	gopkg.in/yaml.v3 => gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
+	gopkg.in/yaml.v3 => gopkg.in/yaml.v3 v3.0.1
```

Comment:
this update lgtm, it includes three bugfixes: https://github.com/go-yaml/yaml/compare/496545a6307b...f6f7691b1fdeb513f56608cd2c32c51f8194bf51